src/ClassFileLocator.php

```diff
@@ -122,6 +122,17 @@ public function accept()
                     if ($i > 0 && is_array($tokens[$i - 1]) && $tokens[$i - 1][0] === T_DOUBLE_COLON) {
                         break;
                     }
+
+                    // ignore anonymous classes on PHP 7.1 and greater
+                    if (&& $i >= 2
```

Comment:
Ha, hehe!

Well, I guess trying to run 

```php
$tokens   = token_get_all($contents);
```

when `$contents` contains usage of anonymous classes will already fail on a PHP version less than PHP 7.1, what do you think?